documentation/source/testbench_tools.rst

```diff
@@ -7,8 +7,57 @@ Logging
 
 Cocotb extends the python logging library. Each dut, monitor, driver, and scoreboard (as well as any other function using the **coroutine** decorator) implements it's own logging object, and each can be set to it's own logging level.
```

Comment:
Typos (2x): it's
(I realize this is not in your changes, but maybe you could just fix it while you are at it?)